src/plugins/intel_cpu/src/compiled_model.h

```diff
@@ -0,0 +1,104 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include <map>
+#include <memory>
+#include <string>
+#include <unordered_map>
+#include <vector>
```

Comment:
Looks that not all include are needed, for example, `map` and `unordered_map`. Please double-check them and other files.

Reply:
ok, new plugin api has removed some old data members, let's also remove the same header files, thansk!

Reply:
done